docs/jsonapi-spec.md

```diff
@@ -592,6 +593,87 @@ See [Multi-Document Failure Considerations](#multi-document-failure-consideratio
 
 If an error occurs the command will not return `data`.
 
+
+### findCollection Command
+
+`findCollection` returns all valid collection tables from a given namespace.
+
+*Syntax:*
+
+```bnf
+<findCollection-command> ::= findCollection <findCollection-command-payload> 
+<findCollection-command-response> ::= <paginated-document-response> 
```

Comment:
reponse only set `status.collections: ["col1", "col2"]` that's all

Reply:
fixed in next commit